compiler/rustc_session/src/lint/builtin.rs

```diff
@@ -1870,6 +1870,16 @@ declare_lint! {
     "detects code samples in docs of private items not documented by rustdoc"
 }
 
+declare_lint! {
+    /// The `invalid_html_tags` lint detects invalid HTML tags. This is a
+    /// `rustdoc` only lint, see the documentation in the [rustdoc book].
```

Comment:
I think there should be a hyphen here:

```suggestion
    /// `rustdoc`-only lint, see the documentation in the [rustdoc book].
```

Reply:
Also, maybe skip the inline code style since you don't use it for "... in the rustdoc book"?

Reply:
I used the same style as the other rustdoc lints, so I'll keep it.